contracts/modules/token/TokenController.sol

```diff
@@ -206,21 +207,63 @@ contract TokenController is ITokenController, LockHandler, LegacyMasterAware {
   /// Function used to claim all pending rewards in one tx. It can be used to selectively withdraw
   /// rewards.
   ///
-  /// @param batchSize  The maximum number of iterations to avoid unbounded loops
-  /// @param batchSize  The maximum number of iterations to avoid unbounded loops
+  /// @param forUser          The address for whom the governance and/or assessment rewards are
+  ///                         withdrawn.
+  /// @param fromGovernance   When true, governance rewards are withdrawn.
+  /// @param fromAssessment   When true, assessment rewards are withdrawn.
+  /// @param batchSize        The maximum number of iterations to avoid unbounded loops when
+  ///                         withdrawing governance and/or assessment rewards.
+  /// @param stakingPools     The addresses of the staking pools that have withdrawable rewards.
+  ///                         When empty, no staking rewards are withdrawn.
+  /// @param stakingTokenIds  The ids of the tokens corresponding to every staking pool provided
+  ///                         above which have withdrwable rewards.
+  /// @param stakingTermIds   The ids of the terms corresponding to every token provided above
+  ///                         which have withdrwable rewards.
   function withdrawPendingRewards(
     address forUser,
     bool fromGovernance,
     bool fromAssessment,
-    uint batchSize
+    uint batchSize,
+    address[] calldata stakingPools,
+    uint[][] calldata stakingTokenIds,
+    uint[][][] calldata stakingTermIds
```

Comment:
I'd try to avoid structs in params as long as it's not solving something such as a stack too deep error. Structs in params come with an accessibiliy cost and some encoding overhead when integrating with these functions. When a user has to use etherscan for various reasons such as a bug in the UI or the wallet not working with our UI, they need to know the inside baseball of these structs. 

Here's a snippet from Uniswap's SwapRouter on etherscan:
![image](https://user-images.githubusercontent.com/2974517/169288924-d22dcee5-2b45-4c9b-afbf-16e0cdf45d84.png)
![image](https://user-images.githubusercontent.com/2974517/169288988-f590b50e-9e14-4fa4-a779-759a46334bde.png)
![image](https://user-images.githubusercontent.com/2974517/169289038-dc40eb82-45f7-4149-8dbb-08e4542ac234.png)

Without the ability to read code, the user would have no idea what the "tuple" should be. My take here is to avoid doing this in user-facing functions as it could cause frustrating experiences.